libvips/foreign/svgload.c

```diff
@@ -273,8 +273,8 @@ vips_foreign_load_svg_parse( VipsForeignLoadSvg *svg, VipsImage *out )
 			 * cairo instead.
 			 */
 			svg->cairo_scale = scale;
-			width = width * scale;
-			height = height * scale;
+			width = VIPS_CEIL(dimensions.em * scale);
+			height = VIPS_CEIL(dimensions.ex * scale);
```

Comment:
I don't like using `dimensions.em` / `.ex` here -- it should be the same on both branches of the `if()`.

I this should be `width = VIPS_CEIL( dimensions.width * scale );`

Reply:
Ohh, I should remember all the C stuff if I'm going to contribute libvips. I was pretty sure that `int * double == int` :)

Fixed